### DISCUSSION TITLE
Responsive styles for size prop

### DISCUSSION THREAD

**Discussion by hmargaryan** [mantinedev/mantine#2953] [Feature requests]
Hey! It would be great to have the same responsive style for `size` like it's demonstrated [here](https://mantine.dev/styles/style-props/#responsive-styles) for width property... or even expand it to all style props

Example:
```
<Text size={{ base: 220, sm: 120 }}>
  Mantine
</Text>
```

Are there any plans on that?

**Comment by rtivital** [comment]
No, there are no such plans, for Text component you can change font-size with `<Text fz={{ base: 12, sm: 14 }} />`

**Reply by mekan-allanov** [reply]
Hi. Is it possible to change size of Pagination responsively like below. I couldn't find answer from docs.
Pagination size={{ base: "sm", md: "md" }}>
or positioning Group component 
<Group position={{base:"center", md:"right"}} >

Thanks

**Reply by rtivital** [reply]
No, it is not supported
